packages/language-service/index.ts

```diff
@@ -6,5 +6,21 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import * as ts from 'typescript/lib/tsserverlibrary';
 export * from './api';
-export {create, getExternalFiles} from './src/ts_plugin';
+
+let plugin: ts.server.PluginModule;
+
+export function create(info: ts.server.PluginCreateInfo): ts.LanguageService {
+  const bundleName = info.config.ivy ? 'ivy.js' : 'language-service.js';
+  plugin = require(`./bundles/${bundleName}`);
+  return plugin.create(info);
+}
+
+export function getExternalFiles(proj: ts.server.Project): string[] {
+  return plugin?.getExternalFiles?.(proj) || [];
```

Comment:
```suggestion
  return plugin?.getExternalFiles?.(proj) ?? [];
```
nit: nullish coalesce